### PR TITLE
fix: commit function in mall full transaction and mall full transaction deferred

### PR DESCRIPTION
**Pull request by DiegoHBustamante**
This PR includes the following changes:

- Fix Commit functions in mall full transaction and mall full transaction deferred.
- Fix Installments function for configure ViewBag in action form of installments view.
- Fix commitInstallment function for transfer data to form in installments view.
- Fix from action in installments view of mallfulltransaction and mallfulltransactiondeferred for execute commit action.

> Case of success Mall Full Transaction
Captura de Pantalla 2024-06-04 a la(s) 22 17 55

> Case of success Mall Full Transaction Deferred
Captura de Pantalla 2024-06-04 a la(s) 22 19 34

### CODE REVIEW

File: transbank-sdk-dotnet-core-rest-example/Controllers/TransaccionCompleta/MallFullTransactionDeferredController.cs [hunk 1]
```diff
@@ -80,7 +80,7 @@ public ActionResult Commit(String token, String child_commerce_code, String chil
                 null,
                 gracePeriod
             ));
-
+            token ??= TempData["Token"] as string;
```
Comment (mastudillot) [inline transbank-sdk-dotnet-core-rest-example/Controllers/TransaccionCompleta/MallFullTransactionDeferredController.cs]: Can token be non-null here? If it has a token, shouldn't it be left unchanged?

File: transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/installments.cshtml [hunk 2]
```diff
@@ -73,10 +73,10 @@
             <div class="card">
               <div class="card-body">
                 <h6 class="card-title">Formulario confirmación con cuotas</h6>
-                <form action="@ViewBag.commit-endpoint")}" method="POST">
+                <form action="@ViewBag.CommitEndpoint" method="POST">
                   <div class="mb-3">
                       <label for="token" class="form-label">Token</label>
-                      <input type="text" class="form-control" id="token" name="token" value="@ViewBag.Token">
+                      <input type="text" class="form-control" id="token_ws" name="token_ws" value="@ViewBag.TokenWs">
```
Comment (mastudillot) [inline transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/installments.cshtml]: This is necessary?

**Review by mastudillot — COMMENTED** [review]
Check my comments

File: transbank-sdk-dotnet-core-rest-example/Controllers/TransaccionCompleta/MallFullTransactionDeferredController.cs (hunk 1 above)
Comment (DiegoHBustamante) [reply transbank-sdk-dotnet-core-rest-example/Controllers/TransaccionCompleta/MallFullTransactionDeferredController.cs]: Code changes were made to omit this validation.

File: transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/installments.cshtml (hunk 2 above)
Comment (DiegoHBustamante) [reply transbank-sdk-dotnet-core-rest-example/Views/transaccion_completa_mall_deferred/installments.cshtml]: No, changes were made to the controller and changes were made to the view.

**Review by mastudillot — APPROVED** [review]
LGTM

**Review by Matiasnickolas — APPROVED** [review]

**Pull request merged**